pkg/ccl/changefeedccl/kvfeed/scanner.go

```diff
@@ -72,6 +72,13 @@ func (p *scanRequestScanner) Scan(
 	var atomicFinished int64
 	for _, span := range spans {
 		span := span
+
+		currentMaxConcurrentExports := maxConcurrentExportRequests(p.gossip, &p.settings.SV)
```

Comment:
This isn't as dynamic as it could be, since if the value is changed while we're blocked on `exportLim.Begin(ctx)` it won't be read until this loop is woken up.  I figured it was a really minor difference in practice and not worth the complexity, but if there was some way of reacting to changes in config settings then we could have the limit be increased from a parallel routine and it'd be able to actually unblock the `exportLim.Begin` call.

Reply:
That's fine.